UnitedIncome/components
# [APP-4505] InfoBox Component Updates

### Description
> Please provide a brief description of your changes below.

Made slight styling tweaks to the `InfoBox` component based on feedback from Monique. Additionally, added a new `isInactive` mode to the component that allows the component to be displayed as "inactive" aka a lower opacity.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Download the codebase and run locally. I added a new story for the `isInactive` state - ensure that the card color changes appropriately - the cards should look similar to the current grayed-out cards in the draft plan section.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.
<img width="244" alt="Screen Shot 2020-03-19 at 10 47 39 AM" src="https://user-images.githubusercontent.com/8663031/77079935-16ce7f80-69cf-11ea-96a7-d5ab2debd354.png">

### Comments
> Any additional comments that will aid with the review process.

This work is being pushed as part of the JIRA ticket below (rather than being it's own story). It is leftover work that we never got to when the component was first created.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/APP-4505)
